src/cheatsheet.html

```diff
@@ -23,9 +23,9 @@ <h2 class="page-header">Every Font Awesome {{ site.fontawesome.version }} Icon,
     {% assign sorted_icons = icons | expand_aliases | sort_by:'class' %}
 
     {% for icon in sorted_icons %}
-    <div class="col-md-4 col-sm-6 col-lg-3">
```

Comment:
@talbs please take another look

I could add a full grid for the print layout, but at the moment we only need the col-4 helper